website/src/blogs/inspired-by-scale17x-e2-80-8a--e2-80-8aa-short-history-of-data-storage.md

```diff
@@ -31,7 +31,9 @@ Regarding #2 — *where did the idea come from* — when trying to answe
 We put together a few slides that offer a light-hearted approach to answering this question.
 
 **The Era of Slow Disk Drives and Scale-up Architectures**
-![https://cdn-images-1.medium.com/max/800/0*6SPflCuTGC2rWXmH](https://lh3.googleusercontent.com/5sIOx2RN3WtVDqKMxZBCb7KhQRudb4Y7p7SlJ8RmR9JAqw5qLuAQRsoZePf21-e6NkA6_8Eo2oZIudCtqYTICYVj42C10H-Px6NUTaGyQwO1UjEcdXxwaRVdmijtMi7bCL4LuHG3)
+
+![https://cdn-images-1.medium.com/max/800/0*6SPflCuTGC2rWXmH](https://cdn-images-1.medium.com/max/800/0*6SPflCuTGC2rWXmH)
```

Comment:
Let's try to understand what the image says and accordingly let's set the alt text for that image.